docs/getting_started.md

```diff
@@ -189,6 +189,33 @@ Assume that you are located at `$MMACTION2` and have already downloaded the chec
         PATH_TO_FRAMES/ LABEL_FILE --use-frames --fps 24 --out-filename demo/demo_out.gif
     ```
 
+### Webcam demo
+
+We provide a demo script to implement read-time action recognition from web camera.
+
+```shell
+python demo/webcam_demo.py ${CONFIG_FILE} ${CHECKPOINT_FILE} ${LABEL_FILE} \
+    [--device ${DEVICE_TYPE}] [--camera-id ${CAMERA_ID}] [--threshold ${THRESHOLD}] \
+    [--sample-length ${SAMPLE_LENGHTH}] [--average-size ${AVERAGE_SIZE}]
+```
+
+Optional arguments:
+- `DEVICE_TYPE`: Type of device to run the demo. Allowed values are cuda device like `cuda:0` or `cpu`. If not specified, it will be set to `cuda:0`.
+- `CAMERA_ID`: ID of camera device If not specified, it will be set to 0.
+- `THRESHOLD`: Threshold of prediction score for action recognition. Only label with score higher than the threshold will be shown. If not specified, it will be set to 0.
+- `SAMPLE_LENGHTH`: Number of frames to be sampled from web camera. If not specified, it will be calculated from the config file by `clip_len x num_clips`.
```

Comment:
length

Reply:
`clip_len` is the key in `SampleFrames` about the pipeline.

Reply:
sample length